CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Adds support for durable entities. Learn more [here](https://learn.microsoft.com
 
 ### Microsoft.DurableTask.Abstractions
 
-- Microsoft.Azure.DurableTask.Core dependency increased to `2.16.0-preview.1`
+- Microsoft.Azure.DurableTask.Core dependency increased to `2.16.0-preview.2`

```

Comment:
According to nuget.org, Microsoft.DurableTask.Abstractions 1.1.0-preview.1 depends on [Microsoft.Azure.DurableTask.Core](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core/) (>= 2.16.0-preview.1). I see that preview.2 is there and also that preview.1 doesn't appear to be listed. Is this expected and okay?

Reply:
Not _ideal_ they don't line up, but it does the right thing in the end. No warnings or restore issues, it picks up `2.16.0-preview.2`. I'll revert this change as it isn't accurate.